### PR TITLE
updates lambda runtime and fixes some warnings

### PR DESCRIPTION
**Pull request by jritsema**
tested with Terraform `v1.1.2`

### CODE REVIEW

File: base/main.tf [hunk 1]
```diff
@@ -13,7 +13,6 @@ terraform {
 # Using the AWS Provider
 # https://www.terraform.io/docs/providers/
 provider "aws" {
-  version = ">= 2.23.0"
```
[the remark this answers is not in the document]
Comment (jritsema) [reply base/main.tf:16]: Hey, no worries.  No problem adding the provider version.  Let me verify which version I tested with, and I will add it.

[the remark this answers is not in the document]
Comment (jritsema) [reply base/main.tf:16]: Does this work for you?

tested with
```
Terraform v1.1.9
on darwin_amd64
+ provider registry.terraform.io/hashicorp/aws v4.14.0
```

**Review by awlawl — APPROVED** [review]
I spun up a test environment using the pr and it had the new versions.

Thanks John!

**Comment by jritsema** [comment]
We should probably cut a release for this.

**Comment by awlawl** [comment]
Yep, released this as v3.0.3. Thanks!

**Pull request merged**